lib/zendesk/client.ex

```diff
@@ -116,6 +173,17 @@ defmodule Zendesk.Client do
     )
   end
 
+  defp add_default_headers(headers, method) when method in [:post, :put] do
```

Comment:
I think we could send the `Content-Type` header with `get`s too as the HTTP specification doesn't preclude it. I seem to recall futzing around with this when working the `zen_ex` library that we'd originally considering using (before we realized that it had been broken for several releases and used like 40,000 different libraries to encode and decode JSON) and observed that sending the header with `get`s had no ill effect. In any case, we could just provide one implementation of `add_default_headers` to reduce surface area 🤷